stable/spring-cloud-data-flow/values.yaml

```diff
@@ -25,6 +25,8 @@ server:
     type: LoadBalancer
     externalPort: 80
     annotations: {}
+  # If not set, use a default configMap else use the configMap from specified value
```

Comment:
please remove this comment as its already defined in the readme

Reply:
Removed